azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+  branches:
+    include:
+      - develop
+      - main
+variables:
```

Comment:
add a space that separates triggers and variables
**and please do the same everywhere necessary**
you can check Projects pipeline as referenec

Reply:
it's done